corehq/apps/commtrack/forms.py

```diff
@@ -128,3 +138,34 @@ def save(self):
                 product._id,
                 val,
             )
+
+
+class LocationTypeStockLevels(forms.Form):
+    """
+    Sub form for configuring stock levels for a specific location type
+    """
+    emergency_level = forms.DecimalField(
+        label=ugettext_noop("Emergency Level (months)"),
+        required=True,
+    )
+    understock_threshold = forms.DecimalField(
+        label=ugettext_noop("Low Stock Level (months)"),
+        required=True,
+    )
+    overstock_threshold = forms.DecimalField(
+        label=ugettext_noop("Overstock Level (months)"),
+        required=True,
+    )
```

Comment:
Yeah, at the top of this same file, as part of the `CommTrackSettingsForm`.  I don't think we're removing that old usage, at least not yet, so this is basically that same stuff pulled out, with the prefixes removed.